contracts/DomainRegistry.sol

```diff
@@ -0,0 +1,89 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.20;
+
+/// @title Domain Registry Contract
+/// @notice This contract allows users to register and manage top-level domain names
+/// @author developed by Ivan Myasoyedov
+contract DomainRegistry {
+    /// @notice Address of the contract owner
+    address public owner;
+
+    /// @notice Fee required to register a domain
+    uint256 public registrationFee;
+
+    /// @dev Struct to store information about a domain
+    struct Domain {
```

Comment:
line 15 
this Struct is a little bit over -logic for this task 
you do not need to store timestamp, iit is already stored in the block
// - 1 point